matter/si91x/support/hal/rsi_hal_mcu_m4.c

```diff
@@ -75,7 +75,7 @@ int soc_pll_config(void) {
   return 0;
 }
 
-void sl_si91x_button_isr(uint8_t pin, uint8_t state) {
+void sl_si91x_button_isr(uint8_t pin, int8_t state) {
```

Comment:
So except for this change the rest now seems to be only clang white space formatting.
However Why is the `state` changed to int8_t? State here seems to be a `bool` and not a negative value why int* is chosen?

Reply:
Yeah I agree, the type bool is relevant, but to match with wifi-sdk prototype making it as int8_t 
https://stash.silabs.com/projects/REDPINE/repos/wifi_sdk/browse/components/device/silabs/si91x/mcu/drivers/hardware_drivers/button/inc/sl_si91x_button.h?at=v3.3.3-fc#193

Reply:
ticket for wifisdk thanks
https://jira.silabs.com/browse/RSCDEV-31323